disperser/apiserver/server_v2.go

```diff
@@ -336,7 +433,7 @@ func (s *DispersalServerV2) GetPaymentStateForAllQuorums(ctx context.Context, re
 	reservationQuorumIds := []core.QuorumID{}
 	reservationCurrentPeriods := []uint64{}
 	if err != nil {
-		s.logger.Debug("failed to get onchain reservation, use zero values", "err", err, "accountID", accountID)
+		s.logger.Error("failed to get onchain reservation, use zero values", "err", err, "accountID", accountID)
```

Comment:
Sorry I wasnt sure what the logic was doing here. These should be `Warn`ings, not errors if we just use a default value when it fails and keep going. An error is when an error is returned immediately.

Reply:
update [47c0368](https://github.com/Layr-Labs/eigenda/pull/1664/commits/47c0368968b84839994e92e3f753c8ccd14f90e2)